pyoverkiz/models.py

```diff
@@ -512,6 +512,7 @@ def __init__(
         **_: Any,
     ) -> None:
         self.id = gateway_id
+        self.gateway_id = gateway_id
```

Comment:
Hi, thank you very much for your contribution. 
I believe this last change has been fixed in #337. Did you rebase your project on the last version of pyoverkise?

Reply:
I did rebase before submitting the PR but my upstream was not in sync.
This is now fixed.
Thanks for your review.